CHANGELOG.md

```diff
@@ -1,11 +1,22 @@
-#Changelog
+# Changelog
 All notable changes to this project will be documented in this file.
-s
+
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [Unreleased]
+### Changed
+- Contact information and bangalore office address have been updated in the contact page.
```

Comment:
- ~bangalore~ -> "Bangalore"
- Contact should be capitalised too

Reply:
@moosicalbovine I have updated the changes in the changelog as per the comments.